docs/install.rst

```diff
@@ -164,6 +166,12 @@ There may be a delay of a day or two between when a new version of ``astropy``
 is released and when a package is available for Anaconda. You can check
 for the list of available versions with ``conda search astropy``.
 
+To install all of the optional dependencies using ``conda`` enter the following::
+
+    conda install -c pkgs/main -c conda-forge \
```

Comment:
I think `defaults` is the user facing name for the main channel.

Reply:
I was thinking that users can [override the default channels](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#default-channels-default-channels), but I guess if they have gone to that length then they know what they are doing.

Reply:
I'll change it.